src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+    <RootNamespace>Microsoft.ML.AutoML</RootNamespace>
+    <IncludeInPackage>Microsoft.ML.AutoML</IncludeInPackage>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <MlDotNetPackageVersion>1.0.0</MlDotNetPackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="$(MlDotNetPackageVersion)" />
+    <PackageReference Include="Microsoft.ML.LightGBM" Version="$(MlDotNetPackageVersion)" />
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="$(MlDotNetPackageVersion)" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
You don't need `GenerateDocumentationFile` here, since it is defaulted to true in https://github.com/dotnet/machinelearning/blob/9a80b78d7aba8463859975119e697afb72f91c09/src/Directory.Build.props#L6.

Reply:
done